.github/workflows/test-suite.yml

```diff
@@ -54,12 +51,22 @@ jobs:
       - if: false
         run: echo jobs need steps
 
+  day_of_week:
```

Comment:
instead of creating a new job day_of_week, maybe we can repurpose `versions` job and name it more generic like `variables` or similar and output not only matrix info but also day of week

Reply:
Great idea! Moved in e43c0c2966ab05aaf9c2a49a6f11bd741c6ada60